public_html/wp-content/plugins/wc-post-types/js/src/speaker/panel-info.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { PluginDocumentSettingPanel } from '@wordpress/edit-post';
+import { TextControl } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import usePostMeta from '../components/hooks/use-post-meta';
+
+export default function SpeakerInfoPanel() {
+	const [ email, setEmail ] = usePostMeta( '_wcb_speaker_email', '' );
+	const [ username, setUsername ] = usePostMeta( '_wcpt_user_name', '' );
+
+	return (
+		<PluginDocumentSettingPanel
+			name="wordcamp/speaker-info"
+			className="wc-panel-speaker-info"
+			title={ __( 'Speaker Info', 'wordcamporg' ) }
+		>
```

Comment:
```suggestion
			title={ __( 'Speaker Info', 'wordcamporg' ) }
			initialOpen={ true }
		>
```

I'm not actually sure off the top of my head if `PluginDocumentSettingPanel` supports this parameter, but I think it would be useful to have the panel start open so it's more discoverable, especially for users with prior experience with the old meta boxes.

Reply:
It doesn't, unfortunately — whether the panel is open or not is a user setting (persisted from the last state).